src/components/TokenPricesTicker.tsx

```diff
@@ -21,21 +22,27 @@ const TokenPricesTicker = ({ ...boxProps }: Props) => {
     [body2, colors],
   )
 
-  const { currentData: tokenPrices } = useGetTokenPricesQuery(
-    { tokens: 'helium,solana', currency },
-    {
-      refetchOnMountOrArgChange: true,
-    },
-  )
+  const [triggerGetTokenPrices, { isFetching, data: tokenPrices }] =
+    useLazyGetTokenPricesQuery({
+      pollingInterval: 1000 * 60,
+      refetchOnReconnect: true,
+      refetchOnFocus: true,
+    })
+
+  useAsync(async () => {
+    await triggerGetTokenPrices({ tokens: 'helium,solana', currency }, false)
```

Comment:
So this triggers it to keep polling? Does it automatically stop when the page loses focus? Or do we want it to keep running always when the app is open, I think that might still be ok?

Reply:
The second argument is used to tell redux if you want to keep the cache or not. Polling is set to 1 minute on the hook in this file